yeagerai/genlayer-simulator
# 171 simulator   e2e frontend tests for contract examples

This includes the first version of e2e test with setup and first test
*How to test:*
- We should have all containers running  except the front end container
- if we run the local frontend, to update the contracts examples with latest please run `npm run cp-examples`
- run `nvm use && npm ci`
- run `npm run test:e2e`
